app/grants/models.py

```diff
@@ -1824,3 +1866,21 @@ class GrantStat(SuperModel):
 
     def __str__(self):
         return f'{self.snapshot_type} {self.created_on} for {self.grant.title}'
+
+
+class GrantCLRCalculation(SuperModel):
+
+    latest = models.BooleanField(default=False, db_index=True, help_text="Is this calc the latest?")
+    grant = models.ForeignKey(Grant, on_delete=models.CASCADE, related_name='clr_calculations',
+                              help_text=_('The grant'))
+    grantclr = models.ForeignKey(GrantCLR, on_delete=models.CASCADE, related_name='clr_calculations',
```

Comment:
@owocki why do both of these have the same related name ?

Reply:
bc they are both `clr_calculations`, just reference from a diff spot.